packages/odyssey-react/src/components/Tabs/Tabs.module.scss

```diff
@@ -29,11 +29,30 @@
   color: var(--TabTextColor);
   font-family: inherit;
   font-size: var(--TabFontSize);
-  font-weight: var(--TabFontWeight);
```

Comment:
Wassup with these Tab changes?

Reply:
I think you won't see this in my PR once I point to `develop`

Reply:
Ah, just realized this was a feature-to-feature PR! 😅